source/boot_security.c

```diff
@@ -11,64 +11,86 @@
 * Unauthorised copying of this file, via any medium is strictly prohibited.
 *
 ******************************************************************************/
-
-
- /******************************** VARIABLES ***********************************/
-
- //static const uint8_t Decryption_Key[DECRYPTION_KEY_LEN_IN_BYTES];
- // uint8_t* input = package->image;
- // uint8_t* output = input;
+ 
 
  
 /********************************* INCLUDES ***********************************/
 #include "boot_security.h"
+
 #include "aes.h"
+
 #include "hal_log.h"
 
- /******************************* MACRO DEFINITIONS ******************************/
-/*
-* Symmetric Key Length for Decryption
-* We use AES-256 for encryption/decryption
-*/
+
+/******************************* MACRO DEFINITIONS ******************************/
+
+    /*
```

Comment:
It seems there are spaces/tabs before the comment lines, while the comments belong to DECRYPTION_KEY_LEN_IN_BITS definition, they need to be vertically aligned with the DECRYPTION_KEY_LEN_IN_BITS definition. 

Please also remove the extra empty lines after the comment